packages/modals/src/elements/DrawerModal/DrawerModal.tsx

```diff
@@ -49,12 +58,32 @@ const DrawerModalComponent = forwardRef<HTMLDivElement, IDrawerModalProps>(
       useModal({
         idPrefix: id,
         modalRef,
-        focusOnMount,
-        restoreFocus,
+        focusOnMount: false /* [1:b] */,
+        restoreFocus: false /* [1:b] */,
         environment,
         onClose
       });
 
+    /* [1:c] */
+    useEffect(() => {
+      if (environment) {
+        if (modalRef.current && isOpen) {
+          if (restoreFocus === undefined ? true : restoreFocus) {
```

Comment:
nit: `restoreFocus === undefined || restoreFocus` also works

Reply:
True, good call. We probably wouldn't need this if we use defaultProps though and it would just be `if (restoreFocus)`.